src/main/java/pixelperfect/event/Event.java

```diff
@@ -7,17 +7,28 @@
  *
  */
 public class Event {
```

Comment:
Let's be careful with this class. It might just become a data class packed with lots of functions.